### PR TITLE
Enhanced winrm_cmd module

### PR DESCRIPTION
**Pull request by Yashvendra**
This PR solves the issue #11440 

## Verification

List the steps needed to make sure this thing works

- [x] Start `msfconsole`
- [x] `use auxiliary/scanner/winrm/winrm_cmd.rb`
- [x] `set RHOSTS [IP]`
- [x] `set USERNAME [USERNAME]`
- [x] `set PASSWORD [PASSWORD]`
- [x] `set CMD [WINDOWS COMMAND]`
- [x] `run`

### CODE REVIEW

File: modules/auxiliary/scanner/winrm/winrm_cmd.rb [hunk 1]
```diff
@@ -38,7 +38,7 @@ def run_host(ip)
     streams = winrm_run_cmd(datastore['CMD'])
     return unless streams.class == Hash
     print_error streams['stderr'] unless streams['stderr'] == ''
-    print_good streams['stdout']
+    print_good "#{ip}:#{rport} => " + streams['stdout']
```
Comment (h00die) [inline modules/auxiliary/scanner/winrm/winrm_cmd.rb]: `peer` may be registered here, try: `"#{peer} => "`.  If it doesn't work, thats fine.

Here you did string interpolation, then string concatenation.  I would recommend switching to just interpolation: `#{ip}:#{rport} => #{streams['stdout']}` (change out ip/rport if peer works)

File: modules/auxiliary/scanner/winrm/winrm_cmd.rb [hunk 2]
```diff
@@ -38,7 +38,7 @@ def run_host(ip)
     streams = winrm_run_cmd(datastore['CMD'])
     return unless streams.class == Hash
     print_error streams['stderr'] unless streams['stderr'] == ''
-    print_good "#{ip}:#{rport} => " + streams['stdout']
+    print_good "#{peer}=> #{streams['stdout']}"
```
Comment (Yashvendra) [inline modules/auxiliary/scanner/winrm/winrm_cmd.rb]: `#{peer}` prints some extra spaces after ip:port in the output.

**Comment by Yashvendra** [comment]
Why is it taking so much time to complete the checking, its been 2 days.

**Comment by bcoles** [comment]
> 
> 
> Why is it taking so much time to complete the checking, its been 2 days.

It's fine. The tests passed. GitHub doesn't realise for some reason.

**Comment by bcoles** [comment]
@msjenkins-r7 test this please

**Comment by busterb** [comment]
## Release Notes

This updates the winrm_cmd aux module to always log the command output.

**Comment by busterb** [comment]
Added e67e2a5477. Thanks @Yashvendra

**Comment by mkienow-r7** [comment]
I noticed the associated issue is still open, likely because one of the [keywords](https://help.github.com/en/articles/closing-issues-using-keywords) wasn't used in front of the issue number. Would someone please confirm if this PR resolved #11440?

**Comment by busterb** [comment]
Thanks @mkienow-r7, I'd say it does the right thing.

**Pull request merged**